scrapy/core/downloader/handlers/aiohttp.py

```diff
@@ -0,0 +1,85 @@
+import asyncio
+from typing import Self
+
+import aiohttp
+from twisted.internet.defer import Deferred
+
+from scrapy import responsetypes
+from scrapy.crawler import Crawler
+from scrapy.http.headers import Headers
+from scrapy.http.request import Request
+from scrapy.http.response import Response
+from scrapy.settings import Settings
+from scrapy.spiders import Spider
+from scrapy.utils.reactor import (
+    set_asyncio_event_loop,
+    verify_installed_reactor,
+)
+
+
+class AiohttpDownloadHandler:
+    def __init__(self, settings: Settings, crawler: Crawler):
+        verify_installed_reactor(
+            "twisted.internet.asyncioreactor.AsyncioSelectorReactor"
+        )
```

Comment:
I think this should follow https://docs.scrapy.org/en/latest/topics/asyncio.html#enforcing-asyncio-as-a-requirement